src/wallet/rpcwallet.cpp

```diff
@@ -3449,10 +3449,7 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
     CWallet* const pwallet = wallet.get();
 
     if (pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS) && !want_psbt) {
-        if (!pwallet->chain().rpcEnableDeprecated("bumpfee")) {
-            throw JSONRPCError(RPC_METHOD_DEPRECATED, "Using bumpfee with wallets that have private keys disabled is deprecated. Use psbtbumpfee instead or restart bitcoind with -deprecatedrpc=bumpfee. This functionality will be removed in 0.22");
-        }
-        want_psbt = true;
+        throw JSONRPCError(RPC_METHOD_DEPRECATED, "bumpfee is not available with wallets that have private keys disabled. Use psbtbumpfee instead.");
```

Comment:
Might be good to move the assert_raises_rpc_error test into test/functional/wallet_bumpfee.py.

Reply:
Done